community/cloud-foundation/templates/pubsub/tests/integration/pubsub.bats

```diff
@@ -4,19 +4,19 @@ source tests/helpers.bash
 
 TEST_NAME=$(basename "${BATS_TEST_FILENAME}" | cut -d '.' -f 1)
```

Comment:
The reason we didn't add pushEndpoint example and corresponding test is because it's very tricky to get the URL for such endpoint. Template configuration is basically one line, but we won't be able to create it unless pushEndpoint's URL:
1) exists
2) is valid HTTPS
3) hosted in a domain that we own
4) domain was registered at project settings
5) or, pushEndpoint is GAE app.
The requirements are [listed here](https://cloud.google.com/pubsub/docs/push#configuring-http-endpoints).

We can add one more Pub/Sub example, whose pushEndpoint value is set to `<FIXME:pushEndpoint>`, but I doubt the value it provides is worth the number of keystrokes involved into its creation.
What do you think, @fawix ?

Reply:
@sourced-pavel I just sync'ed with Prem, let's do one example about it and leave the test for now. if required we will revisit this later.

Reply:
@fawix , will do